packages/lib/storage/src/new/types.ts

```diff
@@ -7,7 +7,7 @@ export interface StorageAPI {
         pageSize: number,
         offset: number,
     ) => Promise<StorageEnvelopContainer[]>;
-    getHaltedMessages: () => Promise<StorageEnvelopContainer[]>;
+    getHaltedMessages: () => Promise<any[]>;
```

Comment:
Please provide the proper type instead of just returning any. If there is no type yet add one to the types.ts file